src/System.Text.Json/tests/Serialization/TestClasses.cs

```diff
@@ -32,6 +32,25 @@ public enum SampleUInt64Enum : ulong
         Max = ulong.MaxValue
     }
 
+    public struct SampleStruct
+    {
+        public int One { get; set; }
+        public double Two { get; set; }
+    }
+
+    public struct SampleStructWithObject
```

Comment:
nit: Maybe rename this to SampleStructWithClass, especially since we special case `System.Object` in a few places. At first glance, I thought this struct would have the following property:
```C#
SampleStructWithObject
{
   public object SomeObject {get; set;}
   // ... others
}
```

It's probably worth adding such a test case, if one doesn't exist already.

Reply:
Sounds good, but just realized that this class is not used within this PR (Have got exception and filed issue, here only successful test cases added), so maybe remove this class for now

Reply:
> ``` 
> SampleStructWithObject
> {
>    public object SomeObject {get; set;}
>    // ... others
> }
> ```

Good case to add but not gonna add a property of object type in this PR as got  an Exception on parse, commented to similar issue instead

Reply:
I think we should keep the test cases with structs containing other classes, objects, etc. (rather than removing it) and if they are failing for some reason, add an `ActiveIssue` attribute to the test rather than removing it.

Reply:
Well i had added them within SimpleTestStruct.cs as a new property to test, i.e. they had no separate test method, but i can create separate struct and test method accordingly